src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstier/domain/enums/ComplexityFactor.kt

```diff
@@ -12,7 +12,7 @@ enum class ComplexityFactor(val registerCode: String) {
   PUBLIC_INTEREST("RPIR"),
   ADULT_AT_RISK("RVAD"),
   STREET_GANGS("STRG"),
-  TERRORISM("TACT");
+  TERRORISM("RTAO");
```

Comment:
unrelated bug: RTAO, not TACT is the code referenced in the ticket, I misread it at the time.